components/AI/ChatInterface.tsx

```
@@ -0,0 +1,270 @@
+                  }`}>
+                  {message.sender === 'user' ? <User size={16} /> : <Bot size={16} />}
+                </div>
+                <div className={`max-w-[75%] ${message.sender === 'user' ? 'text-right' : 'text-left'}`}>
+                  <div className={`inline-block p-3 rounded-2xl text-sm ${message.sender === 'user'
+                    ? 'bg-orange-500 text-white rounded-br-sm'
+                    : 'bg-gray-100 dark:bg-gray-700 text-gray-800 dark:text-gray-200 rounded-bl-sm'
+                    }`}>
+                    {message.sender === 'ai' ? (
+                      <div
+                        className="whitespace-pre-wrap leading-relaxed"
+                        dangerouslySetInnerHTML={{ __html: formatAIResponse(message.content) }}
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)